### PR TITLE
Added shadow DOM example page

### PR DESCRIPTION
**Pull request by esanzgar**
This page will allow us to investigate if it is possible to capture the
selection event from outside the shadow DOM, and if so, how can we do
it.

### CODE REVIEW

File: dev-server/documents/html/shadow-dom.mustache [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      p {
+        color: white;
+        background-color: #666;
+        padding: 5px;
+      }
+     </style>
+     <p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean ultricies mi vitae est. Mauris placerat eleifend leo.</p>
+  </template>
+  <script>
+    customElements.define('my-paragraph',
```
Comment (lyzadanger) [inline dev-server/documents/html/shadow-dom.mustache:22]: Could you add a brief comment to this script to explain what it's doing? Inline is fine.

File: dev-server/documents/html/shadow-dom.mustache [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+          const shadowRoot = this.attachShadow({mode: 'open'})
+            .appendChild(templateContent.cloneNode(true));
+        }
+      }
+    );
+  </script>
+</head>
+<body>
+  <h2>Shadow DOM example</h2>
+  <my-paragraph></my-paragraph>
```
Comment (lyzadanger) [inline dev-server/documents/html/shadow-dom.mustache:38]: Or...maybe even add some content to the page itself (as HTML) to explain what it's designed to allow testing of?

**Review by lyzadanger — APPROVED** [review]
This is just fine, thank you! I suggested adding a comment or some HTML content to explain the purpose of the page a bit more.

File: dev-server/documents/html/shadow-dom.mustache (hunk 2 above)
Comment (esanzgar) [reply dev-server/documents/html/shadow-dom.mustache:39]: I'll do.

File: dev-server/documents/html/shadow-dom.mustache (hunk 1 above)
Comment (esanzgar) [reply dev-server/documents/html/shadow-dom.mustache:22]: Added in the page itself.

**Pull request merged**